config/lib/app.js

```diff
@@ -4,24 +4,18 @@
  * Module dependencies.
  */
 var config = require('../config'),
-  mongoose = require('./mongoose'),
+  mongooseConfig = require('./mongoose-config'),
   express = require('./express'),
-  chalk = require('chalk');
-
-// Initialize Models
-mongoose.loadModels();
-
-//SeedDB
-if (config.seedDB) {
-  require('./seed');
-}
```

Comment:
This didn't really make sense to me - moved this code to be in the callback after the app is started.